mobile/src/bundles/common/components/input/input.tsx

```diff
@@ -1,70 +1,49 @@
 import React from 'react';
 import {
     type Control,
-    type FieldErrors,
     type FieldPath,
     type FieldValues,
 } from 'react-hook-form';
 import { type TextInputProps } from 'react-native';
 import { TextInput } from 'react-native';
 
-import { Text, View } from '~/bundles/common/components/components';
 import { useFormController } from '~/bundles/common/hooks/hooks';
 import { globalStyles } from '~/bundles/common/styles/global-styles';
 
 import { styles } from './styles';
 
 type Properties<T extends FieldValues> = TextInputProps & {
     control: Control<T, null>;
-    errors: FieldErrors<T>;
-    label: string;
     name: FieldPath<T>;
+    readonly?: boolean;
```

Comment:
Do you use it anywhere?

Reply:
No, I was trying readonly andforgot to delete